scripts/feature_tests/feature_test_vars.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export KUBERNETES_VERSION="v1.21.1"
```

Comment:
Same here kubernetes version should be v1.21.0

Reply:
This PR is on old. I have create a new one https://github.com/metal3-io/metal3-dev-env/pull/670